Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/StreamEffect/StreamEffect.xaml.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using System.Threading.Tasks;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.IO;
+using WinMLSamplesGalleryNative;
+using System.Runtime.InteropServices;
+using System.Threading;
+using Windows.UI.Core;
+
+namespace WinMLSamplesGallery.Samples
+{
+    delegate IntPtr WndProc(IntPtr hWnd, uint msg, IntPtr wParam, IntPtr lParam);
+
+    public sealed partial class StreamEffect : Page
+    {
+        string modelPath;
+        bool isPreviewing = false;
+        IntPtr currentHwnd;
+        IntPtr demoHwnd;
+        Task windTask;
+
+        [DllImport("user32.dll", CharSet = CharSet.Auto, SetLastError = true)]
+        static extern IntPtr GetForegroundWindow();
+        [System.Runtime.InteropServices.DllImport("user32.dll", SetLastError = true)]
+        static extern bool DestroyWindow(IntPtr hWnd);
+
+        public StreamEffect()
+        {
+            this.InitializeComponent();
+            // TODO: Currently the WinMLSamplesGalleryNative component will load
```

Comment:
Is this still the case? Just curious as this is something I didn't run into

Reply:
good to know, looks like i might not need it anymore either. could be worth it to check if any of the other interop samples can remove this dummy call as well.